release-doc-in-process/all_vf.TTL

```diff
@@ -526,6 +527,13 @@ vf:realizationOf a          owl:ObjectProperty ;
         vs:term_status      "unstable" ;
         rdfs:comment        "This economic event occurs as part of this agreement." .
 
+vf:mappableAddress a        owl:ObjectProperty ;
+        rdfs:label          "mappable address" ;
+        rdfs:domain         vf:Location ;
```

Comment:
nothing stops us from defining domain as `geo:SpatialThing`

Reply:
Ok, I changed it all to SpatialLocation in the ttl file; will need to also update in other places once we have this.

Reply:
`geo:SpatialThing` not `vf:SpatialThing`

Reply:
thanks, duh!   fixed